analysis/filter/EdgeFilter/src/main/kotlin/de/maibornwolff/codecharta/filter/edgefilter/ParserDialog.kt

```diff
@@ -0,0 +1,37 @@
+package de.maibornwolff.codecharta.filter.edgefilter
+
+import com.github.kinquirer.KInquirer
+import com.github.kinquirer.components.promptInput
+import de.maibornwolff.codecharta.tools.interactiveparser.ParserDialogInterface
+import mu.KotlinLogging
+
+private val logger = KotlinLogging.logger {}
+
+class ParserDialog {
+    companion object : ParserDialogInterface {
+        private const val EXTENSION = "cc.json"
+        override fun collectParserArgs(): List<String> {
+
+            var inputFileName = KInquirer.promptInput(message = "What is the $EXTENSION file that has to be parsed?")
+            if (inputFileName.substringAfter(".") != EXTENSION)
```

Comment:
I think this will not work for filenames that have multiple dots included, for instance: my.file.cc.json
I suggest to use substringAfterLast instead.

Reply:
After last will pick up only the json and not cc.json. Maybe we do a regex match and make sure cc.json is at the end?

Reply:
Ah ok. I think you can do substringAfterLast(".cc.json")

Reply:
Maybe 🤔 I went with the regex approach as it was clearer to me. Now it will make sure .cc.json is the extension of the file, unrelated to how many `.` the file has 👍